package.json

```diff
@@ -111,12 +111,14 @@
     "@rjsf/bootstrap-4": "^2.3.0",
     "@rjsf/core": "^2.3.0",
     "ajv": "^6.12.3",
+    "apexcharts": "^3.22.1",
```

Comment:
For a `astarte-charts` package with a broader use-case, it could have some valid points:
- More chart types
- More curated looks and default design
- Better zooming/panning for a nice interactivity
- Good customization support for design and data handling
- ApexChart is partnered with FusionCharts, establish data-viz company
- ApexChart supports and mantains frontend integrations such as React, Vue, Angular
However this PR is targeted as a Dashboard enhancement, so you have a solid point not to have two similar dataviz libraries at the same time